build.gradle

```diff
@@ -20,7 +22,7 @@ dependencies {
     api ('com.mesosphere.usi:core') { version { branch = usiBranch } }
     api ('com.mesosphere.usi:core-models') { version { branch = usiBranch } }
     api ('com.mesosphere.usi:mesos-client') { version { branch = usiBranch } }
-    api group: 'org.slf4j', name: 'slf4j-api', version: '1.7.26'
+    api group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
```

Comment:
why the downgrade here? is that version only supported in java 11?

Reply:
I changed the plugin to use its class loader first. This resulted in a version conflict with Jenkins. So I downgraded it. I since dropped it.